ansible.cfg

```diff
@@ -1,3 +1,4 @@
 [defaults]
 inventory           = inventory/inventory
 roles_path          = roles/
+vault_password_file = /inventory/group_vars/.vault_passwd
```

Comment:
I think this will specify a file at the root path of the filesystem. You probably meant a relative path. I'll make a quick edit.